nominees/qgis.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "QGIS",
+  "description": "QGIS is a full-featured, user-friendly, free-and-open-source (FOSS) geographical information system (GIS) that runs on Unix platforms, Windows, and MacOS.",
+  "website": "https://www.qgis.org/en/site/",
+  "license": [
+    {
+      "spdx": "GPL-2.0",
+      "licenseURL": "https://github.com/qgis/QGIS/blob/master/COPYING"
+    }
+  ],
+  "SDGs": [
+    {
+      "SDGNumber": 11,
+      "evidenceText": "QGIS has a lot of features that helps develop beautiful cartography, robust geospatial analysis and create landscape/city design.",
+      "evidenceURL": "https://github.com/qgis/QGIS"
```

Comment:
@Rolikasi could you update the `evidenceURL` to point to one of blog article that talks about [QGIS](https://www.unfpa.org/resources/qgis-digital-cartography-censuses-and-surveys) in the context of SDGs rather than using a Github repo.

Reply:
What do you think about [this project](https://stories.forset.ge/en/article/bishkek-heat-desert-without-trees/) as a evidence URL? I made this visualization with QGIS in 2019

Reply:
@Rolikasi projects are expected to provide supporting links/documentation to support their relevance to a given SDG. Apart from you mentioning you did that visualization in QGIS, there is no mention of the QGIS keyword in that article. That alone makes it harder for readers to connect how QGIS Is connected to the given SDG.